config/locales/en/transition_landing_page.yml

```diff
@@ -40,6 +33,13 @@ en:
           You may also need to exchange your UK driving licence for a licence issued by the EU country where you live.
 
           <a class="govuk-!-font-weight-bold" href="/uk-nationals-living-eu" data-track-category="transition-landing-page" data-track-action="/uk-nationals-living-eu" data-track-label="What you can do now">Check what you must do in the country where you live</a>
+      - row_title: "Businesses that import and export goods"
+        list_block: |
+          From 1 January 2021 you will need to make customs declarations to move goods into and out of the EU. You should:
+          <ul>
+            <li><a class="govuk-!-font-weight-bold" href="/eori" data-track-category="transition-landing-page" data-track-action="/eori" data-track-label="What you can do now">get an EORI number</a> if you do not already have one</li>
```

Comment:
I know you've just moved the code, but I think there should be a full stop after `if you do not already have one`, because there's a full stop on the other li's

Reply:
Interesting, thanks - I think it's the style to not have full stops on bullets, so may have to remove the others. Will check 👍

Reply:
Have removed the other full stop after chatting with the content gang